packages/vite-plugin-svelte/src/utils/options.ts

```diff
@@ -576,9 +618,11 @@ export interface PluginOptions {
 	disableDependencyReinclusion?: boolean | string[];
 
 	/**
-	 * Force Vite to pre-bundle Svelte libraries
+	 * Enable support for Vite's dependency optimization to prebundle Svelte libraries.
 	 *
-	 * @default false
+	 * To disable prebundling for a specific library, add it to `optimizeDeps.exclude`.
+	 *
+	 * @default true
```

Comment:
```suggestion
	 * @default true for dev, false for build
```

maybe we should update here too.